pandas/tests/arrays/sparse/test_astype.py

```diff
@@ -131,3 +133,14 @@ def test_astype_dt64_to_int64(self):
         arr3 = SparseArray(values, dtype=dtype)
         result3 = arr3.astype("int64")
         tm.assert_numpy_array_equal(result3, expected)
+
+
+def test_dtype_sparse_with_specified_fill_value():
+    # GH 49987
+    # specify fill_value which not present in data
```

Comment:
Better to adjust the test name than to leave a comment

CI failures seem legit

Reply:
How about this one? Apparently I'm not very good at naming them.